recipes/mget3/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "mget3" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mget3-{{ version }}.tar.gz
+  sha256: e722414b869f5c7775b9b4472db6a0ca42790ca0706b91b4551cdfa87c8b6621
+
+build:
+  # Our main reason for releasing mget3 on conda-forge is to support users of
+  # ArcGIS Pro 3.2.2 and later. ArcGIS Pro includes Anaconda Python 3.9 or
+  # later and only runs on Windows. So we're skipping Python < 3.9, osx, and
+  # linux. We depend on some packages that don't support Python 3.13 yet, so
+  # skipping Python >= 3.13.
+  skip: true  # [py<39 or py>=313 or py2k or osx or linux]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - git
```

Comment:
Is `git` needed? I didn't see where this is being used.

Reply:
The package uses setuptools_scm to obtain version information, and I believe this requires git. But if you know otherwise, we can remove git.